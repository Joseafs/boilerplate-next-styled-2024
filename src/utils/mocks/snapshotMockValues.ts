export const snapshotYamamotoName = 'Genryusai Shigekuni Yamamoto';
export const snapshotZarakiName = 'Kenpachi Zaraki';
export const snapshotZoroName = 'Roronoa Zoro';

export const snapshotYamamotoDescription =
  'Yamamoto, the Captain-Commander of the Gotei 13, wields the fearsome Zanpakuto Ryujin Jakka. His fiery power and unwavering resolve make him a legend in Soul Society.';
export const snapshotZarakiDescription =
  'Zaraki, captain of the 11th Division, thrives on combat. His wild appearance and unmatched strength have earned him the title of the strongest Shinigami.';
export const snapshotZoroDescription =
  'Zoro, a Straw Hat Pirate, wields three swords and dreams of becoming the world’s greatest swordsman. His scar and lineage connect him to Wano Country’s samurai legacy.';

export const snapshotStringValue = '12.345,67';
export const snapshotFloatNumber = 1234.5678;
