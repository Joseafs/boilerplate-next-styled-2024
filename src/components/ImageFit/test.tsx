import { render } from '@testing-library/react';

import { snapshotYamamotoDescription } from '~/utils/mocks/snapshotMockValues';

import { ImageFit } from '.';

const imagePublicPath = '/images/favicon.png';

describe('ImageFit', () => {
  test('Should match snapshot', () => {
    const { container } = render(
      <ImageFit src={imagePublicPath} description={snapshotYamamotoDescription} width={500} height={500} />,
    );
    expect(container).toMatchSnapshot();
  });
});
