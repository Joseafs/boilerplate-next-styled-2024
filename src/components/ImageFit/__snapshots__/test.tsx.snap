// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImageFit Should match snapshot 1`] = `
<div>
  <img
    alt="Yamamoto, the Captain-Commander of the Gotei 13, wields the fearsome Zanpakuto Ryujin Jakka. His fiery power and unwavering resolve make him a legend in Soul Society."
    class="sc-396cbb3d-0 fJFgPN"
    data-nimg="1"
    decoding="async"
    height="500"
    loading="lazy"
    src="/_next/image?url=%2Fimages%2Ffavicon.png&w=1080&q=75"
    srcset="/_next/image?url=%2Fimages%2Ffavicon.png&w=640&q=75 1x, /_next/image?url=%2Fimages%2Ffavicon.png&w=1080&q=75 2x"
    style="color: transparent;"
    width="500"
  />
</div>
`;
